exercises/kindergarten-garden/canonical-data.json

```diff
@@ -66,43 +91,183 @@
       "cases": [
         {
           "uuid": "149b4290-58e1-40f2-8ae4-8b87c46e765b",
-          "description": "first student's garden",
+          "description": "for Alice, first student's garden",
```

Comment:
I think we said that changing the description was a breaking change. I'm not totally sure though.

Reply:
I think description is mutable as long as it clarifies or fixes typos. Which this does.

Reply:
Agreed. I _think_ we forgot to document this :D

Reply:
We forgot to document this, but I remembered 🥇